pkg/osquery/tables/wifi_networks/wifi_networks.go

```diff
@@ -0,0 +1,125 @@
+// +build windows
+
+package wifi_networks
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/wifi_networks/internal"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+//go:generate go-bindata -nometadata -nocompress -pkg internal -o internal/assets.go internal/assets/
+
+type execer func(ctx context.Context, buf *bytes.Buffer) error
+
+type Table struct {
+	client   *osquery.ExtensionManagerClient
+	logger   log.Logger
+	getBytes execer
+}
+
+func TablePlugin(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	columns := dataflattentable.Columns()
+
+	t := &Table{
+		client:   client,
+		logger:   logger,
+		getBytes: execPwsh(logger),
+	}
+
+	return table.NewPlugin("kolide_wifi_networks", columns, t.generate)
+}
+
+func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+	var results []map[string]string
+	var output bytes.Buffer
+
+	err := t.getBytes(ctx, &output)
+	if err != nil {
+		return results, errors.Wrap(err, "getting raw data")
+	}
+	rows, err := dataflatten.Json(output.Bytes(), dataflatten.WithLogger(t.logger))
+	if err != nil {
+		return results, errors.Wrap(err, "flattening json output")
+	}
+
+	return append(results, dataflattentable.ToMap(rows, "", map[string]string{})...), nil
```

Comment:
I don't know that I've tested it, but I wonder if ToMap would take a nil instead of `map[string]string{})`. Pretty small nit though

Reply:
yep it does take a nil!